netket/operator/_abstract_operator.py

```diff
@@ -123,224 +104,3 @@ def conjugate(self, *, concrete=False) -> "AbstractOperator":
-
-        """
-
-        return self.get_conn_flattened(
-            x.reshape((1, -1)),
-            np.ones(1),
-        )
-
-    def n_conn(self, x, out=None) -> np.ndarray:
-        r"""Return the number of states connected to x.
-
-        Args:
-            x (matrix): A matrix of shape (batch_size,hilbert.size) containing
-                        the batch of quantum numbers x.
-            out (array): If None an output array is allocated.
-
-        Returns:
-            array: The number of connected states x' for each x[i].
-
-        """
-        if out is None:
-            out = np.empty(x.shape[0], dtype=np.intc)
-        self.get_conn_flattened(x, out)
-        out = self._n_conn_from_sections(out)
-
-        return out
-
-    @staticmethod
-    @jit(nopython=True)
-    def _n_conn_from_sections(out):
-        low = 0
-        for i in range(out.shape[0]):
-            old_out = out[i]
-            out[i] = out[i] - low
-            low = old_out
-
-        return out
-
-    def to_sparse(self) -> _csr_matrix:
-        r"""Returns the sparse matrix representation of the operator. Note that,
-        in general, the size of the matrix is exponential in the number of quantum
-        numbers, and this operation should thus only be performed for
-        low-dimensional Hilbert spaces or sufficiently sparse operators.
-
-        This method requires an indexable Hilbert space.
-
-        Returns:
-            The sparse matrix representation of the operator.
-        """
-        concrete_op = self.collect()
-        hilb = self.hilbert
-
-        x = hilb.all_states()
-
-        sections = np.empty(x.shape[0], dtype=np.int32)
-        x_prime, mels = concrete_op.get_conn_flattened(x, sections)
-
-        numbers = hilb.states_to_numbers(x_prime)
-
-        sections1 = np.empty(sections.size + 1, dtype=np.int32)
-        sections1[1:] = sections
-        sections1[0] = 0
-
-        ## eliminate duplicates from numbers
-        # rows_indices = compute_row_indices(hilb.states_to_numbers(x), sections1)
-
-        return _csr_matrix(
-            (mels, numbers, sections1),
-            shape=(self.hilbert.n_states, self.hilbert.n_states),
-        )
-
-        # return _csr_matrix(
-        #    (mels, (rows_indices, numbers)),
-        #    shape=(self.hilbert.n_states, self.hilbert.n_states),
-        # )
-
-    def to_dense(self) -> np.ndarray:
-        r"""Returns the dense matrix representation of the operator. Note that,
-        in general, the size of the matrix is exponential in the number of quantum
-        numbers, and this operation should thus only be performed for
-        low-dimensional Hilbert spaces or sufficiently sparse operators.
-
-        This method requires an indexable Hilbert space.
-
-        Returns:
-            The dense matrix representation of the operator as a Numpy array.
-        """
-        return self.to_sparse().todense().A
-
-    def apply(self, v: np.ndarray) -> np.ndarray:
```

Comment:
also this one can be left here, with a definition of what it does and a NotImplementedError